mapi.py

```diff
@@ -1,91 +1,54 @@
-from flask import Flask, jsonify, request, session, Blueprint
```

Comment:
Hi Carlos, is `session` a module our group wrote? I can't seem to find a module with this name on Google. `Session` is a method in `request`, but that's not what is imported here. Just wondering.

Reply:
`session` is a submodule within `flask`. We didn't write it. See: http://flask.pocoo.org/docs/0.12/api/#flask.session